packages/visx-xychart/src/components/XYChart.tsx

```diff
@@ -34,7 +34,7 @@ export type XYChartProps<
   /** Total height of the desired chart svg, including margin. */
   height?: number;
   /** Margin to apply around the outside. */
-  margin?: Margin;
+  margin?: Margin | null;
```

Comment:
I don't think you need the null here.
You can consider converting it to `Partial<Margin>` to make it consistent?